src/hotspot/share/runtime/vframe.cpp

```diff
@@ -678,10 +678,10 @@ javaVFrame* vframeStreamCommon::asJavaVFrame() {
 
 #ifndef PRODUCT
 void vframe::print() {
-  if (WizardMode) print_on(tty);
+  print_on(tty);
 }
 void vframe::print_on(outputStream* st) const {
-  if (WizardMode) _fr.print_value_on(st,NULL);
```

Comment:
There's a couple other printing function that call vframe::print() so I think that should keep WizardMode on for now until the callers become UL converted.
I don't think the vframe::print_on() version should keep WizardMode since logging calls it with the verbose logging level (guessing this is true, please check!)

Reply:
>You don't need the WizardMode guard here and in print_on. UL is supposed replace WizardMode and Verbose so the correct fix would be to elide it from print_on and only log using print_on when the logging level is logically equivalent to WizardMode (as you have done elsewhere).

It was actually in response to what you said :-). I'm OK with either way.